mediapicker/source-camera/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+    id 'org.jetbrains.kotlin.plugin.parcelize'
+    id 'org.jetbrains.kotlin.kapt'
+    id 'dagger.hilt.android.plugin'
```

Comment:
Suggestion (💡): Although what you did doesn't hurt, consider removing the below unnecessary plugins and dependencies on this newly added module:

`plugins`
```gradle
    id 'org.jetbrains.kotlin.plugin.parcelize'
    id 'org.jetbrains.kotlin.kapt'
    id 'dagger.hilt.android.plugin'
```

`dependencies`
```gradle
    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$kotlinxCoroutinesVersion"

    implementation "com.google.dagger:hilt-android:$gradle.ext.daggerVersion"
    implementation "com.google.dagger:hilt-android-compiler:$gradle.ext.daggerVersion"
    kapt "com.google.dagger:hilt-compiler:$gradle.ext.daggerVersion"
```

Reply:
Good call, thanks! 👍

Reply:
Thanks for this update! 🙇